File: alpaul_gls/Content/Downloadable/PHAO2018092520181018171036/Business/AspNetRol.cs
using System.Collections.Generic;

namespace Business
{
    public class AspNetRol
    {
	    private static Data.AspNetRol obj = new Data.AspNetRol();

        public static int Add(Models.AspNetRol value)
        {
            return obj.Add(value);
        }

        public static int Edit(Models.AspNetRol value)
        {
            return obj.Edit(value);
        }

        public static int Delete(stirng value)
        {
            return obj.Delete(value);
        }

        public static Models.AspNetRol GetById(string ID)
        {
            return obj.GetById(ID);
        }

		public static Models.AspNetRol GetByGUID(string ID)
        {
            return obj.GetByGUID(ID);
        }

        public static List<Models.AspNetRol> GetAll()
        {
            return obj.GetAll();
        }
    }
}

File: alpaul_gls/Content/Downloadable/PHAO2018092520181018161350/Business/AspNetRol.vb
Imports System.Collections.Generic

Public Class AspNetRol
    Public Shared Function Add(value As Models.AspNetRol) As Integer
        Return Data.AspNetRol.Add(value)
    End Function

    Public Shared Function Edit(value As Models.AspNetRol) As Integer
        Return Data.AspNetRol.Edit(value)
    End Function

    Public Shared Function Delete(value As String) As Integer
        Return Data.AspNetRol.Delete(value)
    End Function

    Public Shared Function GetById(value As String) As Models.AspNetRol
        Return Data.AspNetRol.GetById(value)
    End Function

    Public Shared Function GetByGUID(value As String) As Models.AspNetRol
        Return Data.AspNetRol.GetByGUID(value)
    End Function

    Public Shared Function GetAll() As List(Of Models.AspNetRol)
        Return Data.AspNetRol.GetAll()
    End Function
End Class